File: c-plus-plus/URI_1013.cpp
#include <iostream>
using namespace std;
int main ()
{
	int a, b, c;
	cin >> a >> b >> c;
	int ab = (a+b+abs(a-b)) / 2;
	int abc = (ab+c+abs(ab-c)) / 2;
	cout << abc << " eh o maior" << endl;
	return 0;
}

File: ruby/URI_1013.rb
valores = gets.split
a = valores[0].to_i
b = valores[1].to_i
c = valores[2].to_i

maior = (a + b + (a - b).abs) / 2
maior = (maior + c + (maior - c).abs) / 2

puts "#{maior} eh o maior"